pulpcore/cli/core/group.py

```diff
@@ -193,13 +207,13 @@ def role(ctx: click.Context, pulp_ctx: PulpCLIContext, group_ctx: PulpGroupConte
         decorators=[
             group_option,
             click.option("--role", required=True),
-            click.option("--object", "content_object", required=True),
```

Comment:
This should not have been there in the first place, right?

Reply:
Correct, it was  duplicate option.